kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWriteSuiteBase.scala

```diff
@@ -226,6 +226,30 @@ trait DeltaTableWriteSuiteBase extends AnyFunSuite with TestUtils {
       }).map { case protocol: Row => Some(protocol) }.getOrElse(Option.empty)
   }
 
+  /**
+   * Helper method to get the Protocol from the row representation.
+   *
+   * @param row Row representation of the Protocol.
+   * @return the Protocol object
+   */
+  def getProtocolFromRow(row: Row): Protocol = {
```

Comment:
I'd be fine if you just moved this to a static utility method in Protocol.java right above `fromColumnVector`

Reply:
Done.